src/main/java/bio/terra/service/auth/iam/sam/SamIam.java

```diff
@@ -662,13 +662,13 @@ private UserStatusInfo getUserInfoAndVerify(AuthenticatedUserRequest userReq) {
     return userStatusInfo;
   }
 
-  AccessPolicyMembershipV2 createAccessPolicyOneV2(IamRole role, String email) {
-    return createAccessPolicyV2(role, Collections.singletonList(email));
+  AccessPolicyMembershipRequest createAccessPolicyOne(IamRole role, String email) {
+    return createAccessPolicy(role, Collections.singletonList(email));
```

Comment:
Maybe update the `Collections` call while you're here
```suggestion
    return createAccessPolicy(role, List.of(email));
```

Reply:
Oh good call